README.md

```diff
@@ -1 +1,16 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+
+## Author
+
+👨‍💻 **Omar Muhammad**
+
```

Comment:
- [ ] Please specify more sections in your readme file and try to make it look like this [template](https://github.com/microverseinc/readme-template)